### ISSUE TITLE
A11y: Missing aria label for RoomTabRecordingsEditButton

### ISSUE THREAD

**Issue by samuelwei** [THM-Health/PILOS#2906]
**Describe the bug**
Aria label for RoomTabRecordingsEditButton is missing
